### ISSUE TITLE
CircleMarker ShapeOptions Consistency 

### ISSUE THREAD

**Issue by Jhays10** [Leaflet/Leaflet.draw#781]
- [ ] I'm reporting a bug, not asking for help
- [x] I've looked at the [documentation](http://leaflet.github.io/Leaflet.draw/dist/reference-0.4.1.html) to make sure the behaviour is documented and expected
- [x] I'm sure this is a Leaflet Draw code issue, not an issue with my own code nor with the framework I'm using (Cordova, Ionic, Angular, React…)
- [x] I've searched through the issues to make sure it's not yet reported

----

## How to reproduce

- Leaflet version I'm using: 1.0.3
- Leaflet Draw version I'm using: 0.4.12
- Browser (with version) I'm using: Chrome 60
- OS/Platform (with version) I'm using: MacOS 10.12.6

## What behaviour I'm expecting and which behaviour I'm seeing
This is more of an inquiry then a bug. In `Draw.CircleMarker` is it possible to keep the consistency of the use of `shapeOptions`?

Basically I'd expect these lines to be wrapped in a `shapeOptions` object.
https://github.com/Leaflet/Leaflet.draw/blob/master/src/draw/handler/Draw.CircleMarker.js#L12-L18

This would make changing the styles the same between all draw features.

## Minimal example reproducing the issue

- [x] this example is as simple as possible
- [x] this example does not rely on any third party code

Using [jsfiddle](https://jsfiddle.net/ddproxy/mvp7hgou/) or another example site.

**Comment by radumas** [comment]
Related... (I think?) circle-markers don't appear as a draw option in the [documentation](https://leaflet.github.io/Leaflet.draw/docs/leaflet-draw-latest.html#drawoptions)

**Comment by Jhays10** [comment]
Looks as if the documentation might need to be updated. In the code it does look like it's supported.

https://github.com/Leaflet/Leaflet.draw/blob/73cd238820cc11c12af88c8c84a251d4d5ac3dad/src/draw/DrawToolbar.js#L11-L18
